roles/openshift_node_upgrade/tasks/docker/restart.yml

```diff
@@ -0,0 +1,33 @@
+---
+# input variables:
+# - openshift.common.service_type
+# - openshift.common.is_containerized
+# - openshift.common.hostname
+# - openshift.master.api_port
+
+- name: Restart docker
+  service: name=docker state=restarted
+
+- name: Update docker facts
+  openshift_facts:
+    role: docker
+
+- name: Restart containerized services
+  service: name={{ item }} state=started
+  with_items:
+    - etcd_container
+    - openvswitch
+    - "{{ openshift.common.service_type }}-master"
+    - "{{ openshift.common.service_type }}-master-api"
+    - "{{ openshift.common.service_type }}-master-controllers"
+    - "{{ openshift.common.service_type }}-node"
+  failed_when: false
+  when: openshift.common.is_containerized | bool
+
+- name: Wait for master API to come back online
+  wait_for:
+    host: "{{ openshift.common.hostname }}"
+    state: started
+    delay: 10
+    port: "{{ openshift.master.api_port }}"
+  when: inventory_hostname in groups.oo_masters_to_config
```

Comment:
* files/nuke_images.sh
* handlers/main.yml
* tasks/containerized_node_upgrade.yml
* tasks/docker/*
* tasks/rpm_upgrade.yml
* tasks/systemd_units.yml
* templates/*

Most of the files is picked from ``playbooks/common/openshift-cluster/upgrades`` directory. Most likely, most of the files can be removed from that. Need to check each place where they are actually used before that. The remaining files are picked from `openshift_node` role. In the former case, the duplicas can be eventually removed. Just the `openshift_node` files will stay.